tests/irepela/homework_9/test_fraction.py

```diff
@@ -42,6 +43,12 @@ def test_fraction_math_operators():
     assert Fraction(1, 2) + 2 == Fraction(5, 2)
     assert 2 + Fraction(1, 2) == Fraction(5, 2)
 
+    # not valid add
+    with pytest.raises(TypeError):
+        Fraction(1, 2) + "2"
+    with pytest.raises(TypeError):
+        "35" + Fraction(1, 2)
+
```

Comment:
add case when you expect that 2 operand handles the operation

Reply:
Can you provide an example? Cause I'm not sure what do you mean by "2 operand handles the operation"